From 778d03445f41c536f2ca9116ee6acb5f4f01b868 Mon Sep 17 00:00:00 2001
From: Dimitar Petrovski <dimeptr@gmail.com>
Date: Mon, 6 Jul 2020 18:36:31 +0200
Subject: [PATCH] Emit the Initial Fetch Complete signal

BUG: 423845
The plugin does not emit the initialFetchComplete signal which seems
needed by AllContactsMonitor and PersonsModel does not get initialized
since it waitd for the signal from all plugins
---
 src/kpeoplevcard.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/kpeoplevcard.cpp b/src/kpeoplevcard.cpp
index bfaf71e..20531f6 100644
--- a/src/kpeoplevcard.cpp
+++ b/src/kpeoplevcard.cpp
@@ -153,6 +153,8 @@ KPeopleVCard::KPeopleVCard()
 
     processDirectory(QFileInfo(*vcardsLocation));
 
+    emitInitialFetchComplete(true);
+
     connect(m_fs, &KDirWatch::dirty, this, [this](const QString& path) {
         const QFileInfo fi(path);
         if (fi.isFile())
-- 
GitLab

